src/main/java/com/coreos/jetcd/Util.java

```diff
@@ -75,8 +76,15 @@ protected static WatchEvent apiToClientEvent(Event event) {
   /**
    * convert API response header to self defined header.
    */
-  protected static Header apiToClientHeader(ResponseHeader header, long compactRevision) {
+  static Header apiToClientHeader(ResponseHeader header, long compactRevision) {
```

Comment:
I think the original author want to clarify that the conversion is between API header to Custom Header. 
I am fine with remove the API prefix.